modules/ROOT/pages/configuring-components.adoc

```diff
@@ -61,7 +61,7 @@ Components have a lifecycle like any other object in the Mule registry. Lifecycl
 |initialise |The first lifecycle method called once any injectors on the component have been called. This means any properties on the component are set before the initialize lifecycle is called. |`org.mule.api.lifecycle.Initialisable` |`javax.annotation.PostConstruct`
 |start |This is called when the MuleContext is started. |`org.mule.api.lifecycle.Startable` |
 |stop |This is called when the MuleContext is stopped. |`org.mule.api.lifecycle.Stoppable` |
-|dispose |Called as the object is being disposed off. Typically this happens because either the MuleContext is shutting down. |`org.mule.api.lifecycle.Disposible` |`javax.annotation.PreDestroy`
+|dispose |Called as the object is being disposed off. Typically this happens because either the MuleContext is shutting down. |`org.mule.api.lifecycle.Disposable` |`javax.annotation.PreDestroy`
```

Comment:
probably correct but might not be

Reply:
For this one, I checked https://github.com/search?q=org%3Amulesoft+disposable&type=code and saw hundreds of code references. Compared that to https://github.com/search?q=org%3Amulesoft+disposible&type=code - 0 code references.